gui/symfony-docker/src/Controller/EleveurController.php

```diff
@@ -244,6 +247,7 @@ public function disease(Request $request,
     public function specific(ResourceRepository $resourceRepository,
                              $id) : Response
     {
+        
         $animal = $resourceRepository->findOneBy(['id' => $id]);
         if (!$this->eleveurHandler->canHaveAccess($animal, $this->getUser())){
             $this->addFlash('error', 'Ce tag NFC ne correspond pas à un de vos animaux');
```

Comment:
change $this->addFlash('error', 'Ce tag NFC ne correspond pas à un de vos animaux'); 
by $this->addFlash('error', 'Cette ressource ne correspond pas à un de vos animaux');